### PR TITLE
add typescript, cb release test extension

### PR DESCRIPTION
**Pull request by mitchdraft**

### CODE REVIEW

**Comment by mitchdraft** [comment]
note:
- temporarily modifying build to simulate an extension release

after that works, need to restore these conditions:
- only publish extension on release
- restore substitution of package.json version
- use correct extension target (not squash test)

**Comment by mitchdraft** [comment]
notes:
- installing typescript from the hack script did not work
- I added typescript to the go-vsce builder and verified locally that tsc exists in the container
  - now expect the tsc step to go through

**Comment by mitchdraft** [comment]
if this passes, I will restore the true name, then try to make a release

File: cloudbuild.yaml [hunk 1]
```diff
@@ -91,7 +91,7 @@ steps:
   waitFor: ['compile', 'npminstall']
   id: 'push-github-resources'
 
-- name: 'gcr.io/$PROJECT_ID/go-vsce:0.1.3'
+- name: 'gcr.io/$PROJECT_ID/go-vsce:mkdev'
```
Comment (yuval-k) [inline cloudbuild.yaml]: use released version

File: editor/vscode/src/extension.ts [hunk 2]
```diff
@@ -88,6 +89,10 @@ async function getremote(extPath: string): Promise<string> {
     // make sure its the one we expect:
     // first split because the github hash includes the filename
     let hashParts = ks.checksum.split(" ");
+    console.log("ks.checksum");
```
Comment (yuval-k) [inline editor/vscode/src/extension.ts]: remove conole log?

File: editor/vscode/src/extension.ts [hunk 3]
```diff
@@ -170,6 +175,7 @@ class SquashExtension {
     async debug() {
         // run the squashkube binary with -server
 
+        console.log("TESTING");
```
Comment (yuval-k) [inline editor/vscode/src/extension.ts]: remove?

**Pull request merged**